src/pages/organize/[orgId]/campaigns/index.tsx

```diff
@@ -33,14 +79,275 @@ export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
+                <Box flexGrow={ 0 } width={ 0.5 }>
+                    <List disablePadding={ true }>
+                        { unsorted.map(item => (
+                            <ListItem key={ item.key } style={{ background: grey[200], height: '4rem', margin: '1rem 0', paddingLeft:'0.5rem' }}>
+                                <NextLink href="/" passHref>
+                                    <Link color="inherit" variant="subtitle2">
+                                        { `${item.title} ID: ${item.key}` }
+                                    </Link>
+                                </NextLink>
+                            </ListItem>
+                        )) }
+                    </List>
+                </Box>
+                <Box border={ 1 } borderColor={ grey[400] } flexGrow={ 0 } m={ 3 } mt={ 2 } p={ 2 }>
+                    <FormControl component="fieldset">
+                        <FormLabel component="legend">
+                            <Typography color="textPrimary" variant="subtitle1">
+                                <Msg id="pages.organizeAllCampaigns.filter.filter" />
+                            </Typography>
+                        </FormLabel>
+                        <FormGroup>
+                            <FormControlLabel
+                                control={ <Checkbox checked={ filtered.standalones } color="primary" name="standalones" onChange={ handleChange }/> }
+                                label={ intl.formatMessage(
+                                    { id: 'pages.organizeAllCampaigns.filter.standalones' }) }
+                            />
+                            <FormControlLabel
+                                control={ <Checkbox checked={ filtered.surveys } color="primary" name="surveys" onChange={ handleChange } /> }
+                                label={ intl.formatMessage(
+                                    { id: 'pages.organizeAllCampaigns.filter.surveys' }) }
+                            />
+                            <FormControlLabel
+                                control={ <Checkbox checked={ filtered.callAssignments } color="primary" name="callAssignments" onChange={ handleChange } /> }
+                                label={ intl.formatMessage(
+                                    { id: 'pages.organizeAllCampaigns.filter.calls' }) }
+                            />
+                            <FormControlLabel
+                                control={ <Checkbox checked={ filtered.canvasses } color="primary"
+                                    name="canvasses" onChange={ handleChange }
+                                /> }
+                                label={ intl.formatMessage(
+                                    { id: 'pages.organizeAllCampaigns.filter.canvasses' }) }
+                            />
+                        </FormGroup>
+                    </FormControl>
+                </Box>
+            </Box>
+            <SpeedDial
```

Comment:
Can `SpeedDial` have a default action, e.g. when I just click the FAB itself. I think on this screen it should default to creating a new campaign.

Or is that a bad idea?

Reply:
I think it's a great idea, I will implement